nemo/collections/nlp/modules/common/text_generation_utils.py

```diff
@@ -662,10 +662,10 @@ def sample_sequence_batch(
             output = inference_strategy.forward_step(batch, tensor_shape)
 
             if parallel_state.is_pipeline_last_stage():
-                output = output[0]['logits'].float()
+                output = output[0]['logits']  # .float()
```

Comment:
@yidong72 implemented this change. Yi is it ok to not convert the outputs to fp32?

Reply:
For inference, I think it is fine not to use fp32.